src/Microsoft.ML.Data/Transforms/HashTransform.cs

```diff
@@ -440,6 +440,10 @@ private ValueGetter<uint> ComposeGetterOne(IRow input, int iinfo)
                 return ComposeGetterOneCore(GetSrcGetter<ushort>(input, iinfo), seed, mask);
             case DataKind.U4:
                 return ComposeGetterOneCore(GetSrcGetter<uint>(input, iinfo), seed, mask);
+            case DataKind.R4:
+                return ComposeGetterOneCore(GetSrcGetter<Single>(input, iinfo), seed, mask);
```

Comment:
>Single [](start = 57, length = 6)

Let's use the C# keywords, so just as we use `uint` above instead of `UInt8` we should probably use `float` here and `double` below.

Reply:
Here and everywhere you're using `Single` and `Double` I think.

---
In reply to: [210717534](https://github.com/dotnet/machinelearning/pull/684#discussion_r210717534) [](ancestors = 210717534)